webapp/src/sui.tsx

```diff
@@ -1132,4 +1132,48 @@ export class Loader extends UIElement<LoaderProps, {}> {
             {children}
         </div>
     }
+}
+
+export interface WebCamProps {
```

Comment:
WebCam is not an semantic ui component. Could you move it into a different tsx file.

Reply:
sui is reserved for semantic ui components.